### PR TITLE
Update sdk to 1.0.78

### PR DESCRIPTION
**Pull request by Velin92**

### CODE REVIEW

File: ElementX/Sources/FlowCoordinators/UserSessionFlowCoordinator.swift [hunk 1]
```diff
@@ -395,6 +395,10 @@ class UserSessionFlowCoordinator: FlowCoordinatorProtocol {
                     analytics.trackError(context: nil, domain: .E2EE, name: .ExpectedDueToMembership, timeToDecryptMillis: timeToDecryptMs)
                 case .historicalMessage:
                     analytics.trackError(context: nil, domain: .E2EE, name: .HistoricalMessage, timeToDecryptMillis: timeToDecryptMs)
+                case .withheldForUnverifiedOrInsecureDevice:
+                    analytics.trackError(context: nil, domain: .E2EE, name: .RoomKeysWithheldForUnverifiedDevice, timeToDecryptMillis: timeToDecryptMs)
+                case .withheldBySender:
+                    analytics.trackError(context: nil, domain: .E2EE, name: .UnknownError, timeToDecryptMillis: timeToDecryptMs)
```
Comment (Velin92) [inline ElementX/Sources/FlowCoordinators/UserSessionFlowCoordinator.swift:401]: @stefanceriu @pixlwave there seems to be there is no specific case for this one, or should we use the `.RoomKeysWithheldForUnverifiedDevice` also here?

Comment (pixlwave) [reply ElementX/Sources/FlowCoordinators/UserSessionFlowCoordinator.swift:401]: Certainly sounds reasonable to me, might be worth checking with the crypto team to confirm?

**Review by pixlwave — APPROVED** [review]
LGTM 👍

File: ElementX/Sources/FlowCoordinators/UserSessionFlowCoordinator.swift (hunk 1 above)
Comment (Velin92) [reply ElementX/Sources/FlowCoordinators/UserSessionFlowCoordinator.swift:401]: @BillCarsonFr tagging you to be sure if I should use` .RoomKeysWithheldForUnverifiedDevice` for both these cases

Comment (BillCarsonFr) [reply ElementX/Sources/FlowCoordinators/UserSessionFlowCoordinator.swift:401]: For `.withheldBySender` use `OlmKeysNotSentError`

File: ElementX/Sources/Services/Timeline/TimelineItems/RoomTimelineItemFactory.swift [hunk 2]
```diff
@@ -156,6 +156,12 @@ struct RoomTimelineItemFactory: RoomTimelineItemFactoryProtocol {
             case .historicalMessage:
                 encryptionType = .megolmV1AesSha2(sessionID: sessionID, cause: .historicalMessage)
                 errorLabel = L10n.timelineDecryptionFailureHistoricalEventNoKeyBackup
+            case .withheldForUnverifiedOrInsecureDevice:
+                encryptionType = .megolmV1AesSha2(sessionID: sessionID, cause: .withheldForUnverifiedOrInsecureDevice)
+                errorLabel = L10n.timelineDecryptionFailureWithheldUnverified
+            case .withheldBySender:
+                encryptionType = .megolmV1AesSha2(sessionID: sessionID, cause: .witheldBySender)
+                errorLabel = L10n.timelineDecryptionFailureWithheldUnverified
```
Comment (BillCarsonFr) [inline ElementX/Sources/Services/Timeline/TimelineItems/RoomTimelineItemFactory.swift:164]: No here for `withheldBySender` you can use the localazy key `timeline.decryption_failure.unable_to_decrypt`

**Review by BillCarsonFr — CHANGES_REQUESTED** [review]
I clarified what string/code to use for withheldBySender

**Pull request merged**